plugins/devtools-common/report.api.md

```diff
@@ -44,6 +45,12 @@ export const devToolsInfoReadPermission: BasicPermission;
 // @public
 export const devToolsPermissions: BasicPermission[];
 
+// @public (undocumented)
+export const devToolsTaskSchedulerCreatePermission: BasicPermission;
+
+// @public (undocumented)
+export const devToolsTaskSchedulerReadPermission: BasicPermission;
```

Comment:
Hm these permissions should perhaps also go on the actual service later, right? One more thing that maybe should be alpha with a plan of moving it away later